code/modules/surgery/bodyparts/head.dm

```diff
@@ -373,8 +266,8 @@
 	if(override)
 		my_head.override_hair_color = hex_string
 	else
-		my_head.hair_color = hex_string
```

Comment:
Is there a reason you got rid of this?

Reply:
the head inherits from the human so this just makes it less confusing i think